pkg/netpol/eval/exposure.go

```diff
@@ -52,29 +55,38 @@ func (pe *PolicyEngine) generateRepresentativePeers(selectorsLabels []k8s.Single
 
 // extractLabelsAndRefineRepresentativePeers extracts the labels of the given pod object and its namespace and refine matching peers
 // helping func - added in order to avoid code dup. in upsertWorkload and upsertPod
-func (pe *PolicyEngine) extractLabelsAndRefineRepresentativePeers(podObj *k8s.Pod) error {
+func (pe *PolicyEngine) extractLabelsAndRefineRepresentativePeers(podObj *k8s.Pod) {
```

Comment:
please explain in documentation what it means to "refine" -- delete a representative pod if there is a real pod that matches its selectors (matchLabels only, no matchExpression).

Reply:
done